tests/pds/dataservice_test.go

```diff
@@ -98,6 +99,115 @@ var _ = Describe("{DeletePDSPods}", func() {
 	})
 })
 
+var _ = Describe("{UpdatePDSHelmVersion}", func() {
+	steplog := "Validate Data service pods post helm upgrade"
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("UpdatePDSHelmVersion", steplog, pdsLabels, 0)
+	})
+
+	It(steplog, func() {
+		var wlparams pdslib.WorkloadGenerationParams
+		var dataservices []PDSDataService
+
+		steplog = "Install older helm verison"
+		Step(steplog, func() {
+			log.InfoD(steplog)
+			ctx := pdslib.GetAndExpectStringEnvVar("TARGET_KUBECONFIG")
+			target := tc.NewTargetCluster(ctx)
+			isOldversion, err := target.IsLatestPDSHelm(params.InfraToTest.OldPDSHelmVersion)
+			if !isOldversion {
+				err = target.DeRegisterFromControlPlane()
+				log.FailOnError(err, "Target Cluster DeRegisteration failed")
+
+				err = pdslib.RegisterClusterToControlPlane(params, tenantID, true)
+				log.FailOnError(err, "Target Cluster Registeration failed")
+			} else {
+				log.InfoD("Target Cluster is with old pds helm version %s", params.InfraToTest.OldPDSHelmVersion)
+			}
+		})
+
+		steplog = "Deploy and validate data service"
+		Step(steplog, func() {
+			log.InfoD(steplog)
+			deps = DeployInANamespaceAndVerify(params.InfraToTest.Namespace)
+		})
+
+		defer func() {
+			steplog = "Delete created dataservice deployments"
+			Step(steplog, func() {
+				log.InfoD(steplog)
+				for _, dep := range deps {
```

Comment:
same as above

Reply:
we need to log the steps using InfoD, so that steps are logged in aetos dashboard